client/my-sites/checkout/checkout/index.jsx

```diff
@@ -290,48 +298,36 @@ export class Checkout extends React.Component {
 			cartItem = getCartItemForPlan( planSlug );
 		}
 
-		if ( startsWith( this.props.product, 'theme' ) ) {
-			cartMeta = this.props.product.split( ':' )[ 1 ];
+		if ( startsWith( product, 'theme' ) ) {
+			cartMeta = product.split( ':' )[ 1 ];
 			cartItem = themeItem( cartMeta );
 		}
 
-		if ( startsWith( this.props.product, 'domain-mapping' ) ) {
-			cartMeta = this.props.product.split( ':' )[ 1 ];
+		if ( startsWith( product, 'domain-mapping' ) ) {
+			cartMeta = product.split( ':' )[ 1 ];
 			cartItem = domainMapping( { domain: cartMeta } );
 		}
 
-		if ( startsWith( this.props.product, 'concierge-session' ) ) {
+		if ( startsWith( product, 'concierge-session' ) ) {
 			cartItem = ! hasConciergeSession( cart ) && conciergeSessionItem();
 		}
 
-		if (
-			( startsWith( product, 'jetpack_backup' ) || startsWith( product, 'jetpack_scan' ) ) &&
-			isJetpackNotAtomic
-		) {
-			cartItem = jetpackProductItem( product );
-		}
-
-		if (
-			[
-				'jetpack_search',
-				'wpcom_search',
-				'jetpack_search_monthly',
-				'wpcom_search_monthly',
-			].includes( product )
-		) {
+		if ( JETPACK_SEARCH_PRODUCTS.includes( product ) ) {
```

Comment:
@AnnaMag - Tagging you for review to check this code looks okay for Search. I switched to this method as I felt the raw strings were fragile.

Reply:
Thanks!